fbw-common/src/wasm/cpp-msfs-framework/lib/inih/ini.h

```diff
@@ -0,0 +1,592 @@
+// Copyright (c) 2023 FlyByWire Simulations
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
I don't think we should claim any copyright or assert our license on this as it's third party code. The copyright claim and license can be found at https://github.com/metayeti/mINI/blob/master/LICENSE. It may be a good idea to keep third party depedencies in an unambiguous subdirectory like `lib/3rdparty/**`

Reply:
Probably done automatically by my IDE - will look into it and 3rd party libs is a good idea